### PR TITLE
Added s390x support for Java Docker images

### PR DESCRIPTION
**Pull request by haubenr**
Signed-off-by: Dirk Haubenreisser <haubenr@de.ibm.com>

Fixed: #1067 

This PR contains the required changes to enable the Java Docker images for architecture s390x.

### CODE REVIEW

File: debian_packages.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
 # python and java only build on a subset of architectures
 - distros: ["debian11"]
-  archs: ["amd64","arm64"]
```
Comment (loosebazooka) [inline debian_packages.yaml:45]: I think I would just separate out the python and java deps here. It's pretty expensive to download some of these deps, and if python isn't building on s390x we shouldn't be fetching them (we prefetch all deps in CI)

Comment (haubenr) [reply debian_packages.yaml:45]: Sure, I can do that.

Comment (haubenr) [reply debian_packages.yaml:45]: Done with latest commit.

File: debian_packages.yaml [hunk 2]
```diff
@@ -40,15 +40,38 @@
...
+
+# debian 11, limited architectures, python only
 - distros: ["debian11"]
   archs: ["amd64","arm64"]
   packages:
-  # python
     - "dash"
     - "libbz2-1.0"
     - "libcom-err2"
-    - "libcrypt1"  # TODO: glibc library for -lcrypt; maybe should be in cc?
```
Comment (loosebazooka) [inline debian_packages.yaml:51]: I think it's okay to leave this in the python one, duplicates are handled by the system.

**Review by loosebazooka — APPROVED** [review]
just a minor issue and then I can merge.

File: debian_packages.yaml (hunk 2 above)
Comment (haubenr) [reply debian_packages.yaml:51]: Sure, I've re-added the libcrypt1 dependency to the list of Python dependencies in the latest commit.

**Review by loosebazooka — APPROVED** [review]

**Comment by loosebazooka** [comment]
Thanks for contributing. This should trigger a build with the new s390x support

**Pull request merged**